src/lib/ngsiNotify/doNotify.cpp

```diff
@@ -58,12 +59,18 @@ static void doNotifyHttp(SenderThreadParams* params, CURL* curl, SyncQOverflow<S
   char                portV[STRING_SIZE_FOR_INT];
   std::string         endpoint;
   std::string         url;
+  int                 providerLimit  = DEFAULT_PAGINATION_LIMIT_INT;
+  int                 providerOffset = DEFAULT_PAGINATION_OFFSET_INT;
+  ApiVersion          apiVersion     = V1;
```

Comment:
Remove this block and use -1 for `providerLimit` and `provideOffset` in the call to httpRequestSend() below, meaning with <0 value that the parameter hasn't to be included.

Reply:
Fixed in 1e47b07